### PR TITLE
updated with links to content, per my own suggestion

### PR DESCRIPTION
**Pull request by graybeal**
https://github.com/ESIPFed/sweet/issues/198

not sure if I should do a direct commit or pull request...

### CODE REVIEW

**Comment by graybeal** [comment]
@rduerr I think it's temporary, it was working when I started but stopped working halfway through (and not just pyLODE, access to SWEET RDF stopped). I forgot to ticket that, thanks for reminding me.

BTW, it does take a long time to display sweetAll in WebVowl, it's a LOT of imports. But it worked eventually, most of the controls were viable.

**Comment by rduerr** [comment]
Ok, then you have my OK - the rest of the links worked for me, though you are right that WebVowl took forever to load.

File: README.md [hunk 1]
```diff
@@ -7,6 +7,13 @@
 # Introduction
 Official repository for Semantic Web for Earth and Environmental Terminology (SWEET) Ontologies.
 
+[View in pyLODE](https://sweetontology.net/sweetAll) &nbsp;
```
Comment (lewismc) [inline README.md:10]: You are trying to connect to an HTTPS URL... we do not have a cert for sweetontology.net. This needs to be changed to `http://sweetontology.net/sweetAll`

File: README.md [hunk 2]
```diff
@@ -7,6 +7,13 @@
 # Introduction
 Official repository for Semantic Web for Earth and Environmental Terminology (SWEET) Ontologies.
 
+[View in pyLODE](https://sweetontology.net/sweetAll) &nbsp; 
+[View in WebVowl](http://www.visualdataweb.de/webvowl/#iri=https://raw.githubusercontent.com/ESIPFed/sweet/master/src/sweetAll.ttl) &nbsp; 
+[View in ESIP COR](http://cor.esipfed.org/ont?iri=http://sweetontology.net/sweetAll) <br/>
+[View in GitHub](https://github.com/ESIPFed/sweet/blob/master/src/sweetAll.ttl) &nbsp;&nbsp;&nbsp;
```
Comment (lewismc) [inline README.md:13]: Remove `&nbsp;&nbsp;&nbsp; `?

File: README.md [hunk 3]
```diff
@@ -7,6 +7,13 @@
 # Introduction
 Official repository for Semantic Web for Earth and Environmental Terminology (SWEET) Ontologies.
 
+[View in pyLODE](https://sweetontology.net/sweetAll) &nbsp; 
+[View in WebVowl](http://www.visualdataweb.de/webvowl/#iri=https://raw.githubusercontent.com/ESIPFed/sweet/master/src/sweetAll.ttl) &nbsp; 
+[View in ESIP COR](http://cor.esipfed.org/ont?iri=http://sweetontology.net/sweetAll) <br/>
+[View in GitHub](https://github.com/ESIPFed/sweet/blob/master/src/sweetAll.ttl) &nbsp;&nbsp;&nbsp; 
+[View raw RDF](https://raw.githubusercontent.com/ESIPFed/sweet/master/src/sweetAll.ttl) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (lewismc) [inline README.md:14]: Remove `&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `?

File: README.md (hunk 1 above)
Comment (graybeal) [reply README.md]: fixed, sorry, thanks

File: README.md (hunk 2 above)
Comment (graybeal) [reply README.md]: made into table

File: README.md (hunk 3 above)
Comment (graybeal) [reply README.md]: made into table

**Comment by lewismc** [comment]
@rduerr can you change your review to approve? Thanks

**Pull request merged**